### ISSUE TITLE
Install-VsBuildTools2017 not installing even with elevation

### ISSUE THREAD

**Issue by paschott** [RichieBzzzt/PoshSSDTBuildDeploy#18]
I tried to use this to install the VS Build Tools. It downloads the install EXE, but never goes any further. I can run the EXE and install the Build binaries from that file, but not sure why the install isn't working.  Once installed, everything works fine.

**Comment by RichieBzzzt** [comment]
Hello!

Sorry, I have not been getting any messages for this project, despite being admin.

I'm a little confused by what you mean - you say it gets installed but that you're unable to install?

**Comment by paschott** [comment]
Sorry for the delay, Richie. Meant to respond earlier. I get the install downloaded, but never see anything saying that it is installed when I run it as said.  If I manually run the downloaded EXE, it gets installed, but not when I try to run it via the command given.  It might be attempting a silent install, but I'd normally expect some sort of feedback if that were the case.

**Comment by RichieBzzzt** [comment]
Hmm, ok I’ll spin up a vm and have a go. I’ll have to give a look over the code again as I’ve not looked at it in a while. 

I think it does run a silent install, maybe I can get the output to be redirected to the PowerShell console.

**Comment by RichieBzzzt** [comment]
I never got round to solving... I wonder if this would be a better thing to use to install VS instead of my module.

https://github.com/Microsoft/vssetup.powershell

**Comment by paschott** [comment]
I'm open to other ways to do this. My main issue was that when I ran the code it didn't seem to do anything on my machine other than download the installer. Definitely a fan of not re-inventing things just to do them yourself.

**Comment by RichieBzzzt** [comment]
Definitely. I don't remember finding that module back when I started this one, else I would've used that one myself!
